app/views/works/_meta.html.erb

```diff
@@ -56,11 +56,24 @@
       </dd>
     <% end %>
 
-    <dt class="stats"><%= ts("Stats") %>:</dt>
+    <dt class="stats"><%= ts("Stats:") %></dt>
     <dd class="stats">
 <% end %><!-- end of cache -->
 
       <%= work_meta_list(@work, @chapter) %>
+      <% if logged_in_as_admin? %>
```

Comment:
It looks like there are two `<dt>` elements here -- the one on line 68 that contains the actual IP address data should be changed to a `<dd>` (the closing tag on line 74 is correct). Also, `<dt>` and `<dd>` elements can only ever have the `<dl>` element as the direct parent, but right now, their parent is a `<dd>` element. They'll need to be moved outside the stats `<dd>` so their parent element becomes the `<dl>` that surrounds the entire meta section.

For class names, we try to use single words separated by spaces rather than smashing words together. `<dt class="ip">` and `<dd class="ip">` would be sufficient here.

Lastly, there are some indenting issues -- so once you put all the changes in, it should be like this (starting on line 63):
```
  <%= work_meta_list(@work, @chapter) %>
</dd>
<% if logged_in_as_admin? %>
  <dt class="ip">
    <%= ts('IP Address:') %>
  </dt>
  <dd class="ip">
    <% if @work.ip_address.nil? %>
      <%= ts('No address recorded') %>
    <% else %>
      <%= @work.ip_address %>
    <% end %>
  </dd>
<% end %>
```